packages/package_info_plus/package_info_plus_web/lib/package_info_plus_web.dart

```diff
@@ -10,18 +10,31 @@ import 'package:package_info_plus_platform_interface/package_info_platform_inter
 ///
 /// This class implements the `package:package_info_plus` functionality for the web.
 class PackageInfoPlugin extends PackageInfoPlatform {
+  final Client _client;
+
+  /// Create plugin with http client.
+  PackageInfoPlugin(this._client);
+
```

Comment:
Do we really need to have the client injected? We use `get` to see my next comment. any particular reason behind this?

Reply:
I did it to make tests work in this commit: b4b42f3e14656042d03529ce3d031010de283363
Because we need to mock the Client and use mocked HTTP client.